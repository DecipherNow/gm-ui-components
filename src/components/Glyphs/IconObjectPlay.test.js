import React from "react";
import { shallow } from "enzyme";

import IconObjectPlay from "./IconObjectPlay";

describe("IconObjectPlay", () => {
  it("matches snapshot", () => {
    const aIconObjectPlay = shallow(<IconObjectPlay />);
    expect(aIconObjectPlay).toMatchSnapshot();
  });
});