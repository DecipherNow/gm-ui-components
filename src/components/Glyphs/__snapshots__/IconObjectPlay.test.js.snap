// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconObjectPlay matches snapshot 1`] = `
<Icon
  backgroundColor="currentColor"
  backgroundOpacity={1}
  backgroundSizeRatio={1}
  borderColor="currentColor"
  borderOpacity={1}
  borderWidth={1}
  glyph=""
  glyphColor="currentColor"
  glyphName="ObjectPlay"
  glyphSizeRatio={1}
>
  <path
    d="M5.5,5 L18.5,5 C18.7761424,5 19,5.22385763 19,5.5 L19,18.5 C19,18.7761424 18.7761424,19 18.5,19 L5.5,19 C5.22385763,19 5,18.7761424 5,18.5 L5,5.5 C5,5.22385763 5.22385763,5 5.5,5 Z M10.7764082,8.96815382 C10.6944718,8.91379695 10.5983272,8.88480539 10.5,8.88480539 C10.2238576,8.88480539 9.99999999,9.10866301 9.99999999,9.38480539 L9.99999999,14.6151946 C9.99999999,14.7135218 10.0289916,14.8096664 10.0833484,14.8916028 C10.2360044,15.1217131 10.5462979,15.1845022 10.7764082,15.0318462 L14.7184948,12.4166516 C14.7741383,12.3797375 14.8218241,12.3320517 14.8587382,12.2764082 C15.0113942,12.0462979 14.9486051,11.7360044 14.7184948,11.5833484 L10.7764082,8.96815382 Z"
    fillRule="nonzero"
    id="Play-Object"
  />
</Icon>
`;
